libsolidity/lsp/LanguageServer.cpp

```diff
@@ -188,18 +188,20 @@ void LanguageServer::changeConfiguration(Json::Value const& _settings)
 	}
 
 	m_settingsObject = _settings;
-	Json::Value jsonIncludePaths = _settings["include-paths"];
+	Json jsonIncludePaths;
+	if (_settings.contains("include-paths"))
+		jsonIncludePaths = _settings["include-paths"];
```

Comment:
```suggestion
	Json jsonIncludePaths = _settings.contains("include-paths") ? _settings["include-paths"] : json::object();
```
or so would also seem more consistent to me with the other cases like this - is there a difference in behaviour between those?
Does ``Json().empty()`` throw or just return ``true``?

Reply:
`Json::empty()` is `noexcept`.